### DISCUSSION TITLE
Trouble logging into CouchDB after restarting

### DISCUSSION THREAD

**Discussion by JeffreyQin** [apache/couchdb#4708] [Q&A] [answered]
I recently installed CouchDB on my computer and I have been experiencing issues logging in. As a MacOS user, I found the ini file located at 

`Library/Preferences/couchdb_local.ini`

where I modified the admin section so it looks like

```
[admins]
username = randomusername
password = randompassword
```

After restarting my computer and entering the file again, I can see that both the username and password became hashed, looking something like

```
[admins]
username = -pbkdf2-dadsiada3f2w34f,jjfioji3jf3,10
password = -pbkdf2-f4i32f0j230jf03,3fijf203jf0,10
```

However, I'm still having issues logging into the Fauxton portal at localhost:5984/_utils both using the hashed and unhashed credentials. 

Is there anything I have done wrong? Thanks.

**Comment by rnewson** [comment, marked as answer]
Yes, you have done something wrong, the entries are `username = password`

so

```
[admins]
foo = bar
```

you can then login as user foo with password bar.
